### ISSUE TITLE
Assets do not exist

### ISSUE THREAD

**Issue by jameslkingsley** [sbine/nova-route-viewer#1]
Assets do not exist as there is no `dist` folder when cloned into `vendor`.

**Comment by fishmad** [comment]
Yes.

The `/dist` folder is named `/resources` for some reason?
Its causing me server 500 errors:
![image](https://user-images.githubusercontent.com/12012101/44587476-6601fd80-a7f6-11e8-8071-259d2849a905.png)

mix-manifest.json is the package:
```
{
    "/dist/js/tool.js": "/dist/js/tool.js",
    "/dist/css/tool.css": "/dist/css/tool.css"
}

```

**Comment by sbine** [comment]
Thanks, will have this fixed shortly.

**Comment by sbine** [comment]
Fixed in v0.0.1 -- note that I had to revert the package name to `sbine/route-viewer`, so please update your dependencies accordingly.
